nitro.config.ts

```diff
@@ -1,19 +1,21 @@
 //https://nitro.unjs.io/config
 export default defineNitroConfig({
   routeRules: {
-    '/w3s': {
+    '/': {
```

Comment:
Doesn't this not correspond to an existing route?

Reply:
'/' corresponds to "all" 
/w3s route still exists

Reply:
we have:
/w3s 
/mux/upload
/mux/uploadStatus

Reply:
Understood, but why is this necessary?

Reply:
if we want to add more routes in the future we dont have to be verbose about adding 'cors: true' in every route. it was more relevant when i had mux/upload and mux/uploadStatus

Reply:
Ahh I see, for some reason I thought `cors:true` was still present on the individual routes.